src/nn/wasm/NNOps.js

```diff
@@ -2,7 +2,7 @@ if (!global._babelPolyfill) {
 	require('babel-polyfill');
 }
 
-import Module from './nn_ops'
+//import Module from './nn_ops'
```

Comment:
do we need this file anymore?

Reply:
I think we don't need this file anymore since we try to use tfjs-wasm to replace it. I just comment out this line because it will cause the error related to 'fs' module. We can delete some files when tfjs-wasm can totally replace the nn_ops.